2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -97,20 +96,69 @@ Non sono definiti vincoli o limiti sulle memorie primaria e secondaria.
 
 ==== Operazioni
 
-// Dettagliare le operazioni, una volta confermate, secondo quanto descritto da 9.6.4.7
+//non ho molto chiaro cosa si richiedesse in questa sezione, sotto riporto lo standard
+//Specify the normal and special operations required by the user such as:
+// - the various modes of operations in the user organization*/
+Il sistema permette di attuare e visualizzare diverse operazioni di gestione del magazzino, in modo da semplificare l'interazione tra gestori e addetti.
+Il sistema deve permettere all'utente le seguenti operazioni:
+- creazione del magazzino (manualmente o tramite caricamento di un file SVG);
+- modificare le dimensioni dell'ambiente 3D (lunghezza, larghezza, altezza);
+- caricare un file SVG;
+- creare e modificare le dimensione di scaffali (lunghezza e larghezza);
+- eliminare gli scaffali;
+- spostare gli scaffali nell'ambiente;
+- definire il numero di piani dello scaffale;
+- creare e modificare i bin (lunghezza, larghezza e altezza);
+- leggere informazioni riguardo agli scaffali e al contenuto dei bin;
+- eliminare i bin;
+- caricare i dati da database;
+- richiedere lo spostamento di un prodotto da un bin a un altro;
+- ricercare un prodotto nel magazzino;
+- visualizzare ed esplorare il magazzino.
+
+Modifiche alla posizione dei prodotti comporta l'invio di una notifica a gli addetti che provvederanno ad attuare lo spostamento.
```

Comment:
Non è un a ripetizione del punto precedente "richiedere lo spostamento di un prodotto da un bin ad un altro"? Potresti metterlo come sottopunto.

Reply:
Specify the normal and special operations required by the user such as:
 - the various modes of operations in the user organization

ho provato così ad adattare le norme, ovvero: il gestore fa: operazioni, 
che comportano addetto fa: operazioni.
Posso modificare come richiedi, ma come le ho scritte io ho cercato di mettere in evidenza chi utilizza/interagisce con il software e come. Modifico in modo da rendere più chiaro questo aspetto.
Se ancora non ti convince modifico come richiedi.